schemas/primary_diagnosis.json

```diff
@@ -201,21 +188,24 @@
           "Vomiting",
           "Weight Loss"
         ]
-      }
+      },
+      "meta":{}
     },
     {
       "name": "performance_status",
-      "description": "Indicate the donor's performance status at the time of primary diagnosis (ECOG performance score https://ecog-acrin.org/resources/ecog-performance-status).",
+      "description": "Indicate the donor's performance status grade at the time of primary diagnosis (ECOG performance score grades: https://ecog-acrin.org/resources/ecog-performance-status).",
       "valueType": "string",
       "restrictions": {
         "codeList": [
-          "Fully active, able to carry on all pre-disease performance without restriction",
-          "Restricted in physically strenuous activity but ambulatory and able to carry out work of a light or sedentary nature (ie. Light house work, office work)",
-          "Ambulatory and capable of all selfcare but unable to carry out any work activities; up and about more than 50% of waking hours",
-          "Capable of only limited selfcare; confined to bed or chair more than 50% of waking hours",
-          "Completely disabled; cannot carry on any selfcare; totally confined to bed or chair"
+          "0",
```

Comment:
Should this say Grade 0, not just zero?

Reply:
Sure, I can change it to include "Grade". Will it matter if a data submitter submits "grade 0" instead of "Grade 0"?

Reply:
both will be accepted!  Only Grade 0 will be saved as a value though so all saved data is consistent

Reply:
Thanks Rosi, I updated the controlled vocab to include Grade